### PR TITLE
Correct Markdown link error

### PR DESCRIPTION
**Pull request by DazWilkin**
Fixes #(issue-number)

## Proposed Changes

* 
* 
*

### CODE REVIEW

File: install/Knative-with-GKE.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ for getting started:
   deploy one of your choosing.
 
 * To get started by creating a build, see
-  (Creating a simple Knative Build)[../build/creating-builds.md]
+  [Creating a simple Knative Build](../build/creating-builds.md)
```
Comment (samodell) [inline install/Knative-with-GKE.md]: This change fixes the formatting, but the linked page doesn't exist. @RichieEscarez , where did you want this link to go?

Comment (cch0) [reply install/Knative-with-GKE.md]: My guess is that this file is still in the making, see this PR https://github.com/knative/docs/pull/261

**Comment by rgregg** [comment]
Looks like #261 was merged, so this link is now valid.

/lgtm
/approve

**Pull request merged**